### ISSUE TITLE
[Feature] Multi users

### ISSUE THREAD

**Issue by tetraf** [Uzay-G/archivy#65]
It may be useful to have multiple users in Archivy, for a family service or to use inside a company.
Ideally with external identity providers (LDAP, SAML…).

To go further we could think to have different statuses for bookmarks, like: public, private, shared…

**Comment by Uzay-G** [comment]
Indeed, we need to implement login so people can access archivy from a remote server. Linked to https://github.com/Uzay-G/archivy/issues/19
